native/Dockerfile-win-dockcross

```diff
@@ -1,9 +1,10 @@
-FROM dockcross/windows-shared-x64
+ARG IMAGE
+FROM dockcross/$IMAGE
 
 ARG GO_VERSION=1.22.1
 ARG GO_ARCHIVE_CHECKSUM=aab8e15785c997ae20f9c88422ee35d962c4562212bb0f879d052a35c8307c7f
 
-ENV DEFAULT_DOCKCROSS_IMAGE native-windows-cross-compile
+ENV DEFAULT_DOCKCROSS_IMAGE native-${IMAGE}-cross-compile
```

Comment:
@fab-10 is this change related?

Reply:
not required for the feature, but it enables building the linux native lib on Windows so it can be to run the plugin under WSL, will move it in a separate PR